image:
  width: 640
  height: 480
  rectified: true # /image_rect_color면 true 권장


camera_intrinsics:
  K: [422.037858, 0.0, 245.895397,
    0.0, 435.589734, 163.625535,
    0.0, 0.0, 1.0]
  D: [-0.289296, 0.061035, 0.001786, 0.015238, 0.0]
  distortion_model: plumb_bob


# LiDAR→Camera 변환 (카메라 좌표계 = R*LiDAR + t)
# 회전 없음, 평행이동만 (사용자 제공값)
extrinsics:
  T_lidar_camera: [0,1,0,0.45,
    0,0,1, 0.46,
    -1,0,0, 0.9,
    0,0,0, 1.0]


viz:
  clamp_to_image: true
